tests/unit/tasks/test_python_task.py

```diff
@@ -0,0 +1,26 @@
+from griptape.artifacts import BaseArtifact, TextArtifact
+from griptape.tasks import PythonTask, BaseTextInputTask, BaseTask
+from tests.mocks.mock_prompt_driver import MockPromptDriver
+from griptape.structures import Pipeline
+
+
+def hello_world(task: BaseTask) -> BaseArtifact:
+    greeting = "Hello World!"
+    # We don't want to create an artifact, so just pass the parent task's input back
+    return TextArtifact(greeting)
```

Comment:
this comment doesn't mesh with what's being returned. We want to tell customers that if they want essentially a no-op to return `task.input`, right?